<ui version="4.0">
 <class>AswDialog</class>
 <widget class="QDialog" name="AswDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Расчет ASW</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_speed">
     <property name="text">
      <string>Скорость (узлы)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="spin_speed"/>
   </item>
   <!-- Добавьте все поля аналогично (QSpinBox для направления, QDateTimeEdit -->
   <item>
    <widget class="QPushButton" name="btn_cancel">
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_calculate">
     <property name="text">
      <string>Рассчитать</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>